mule-module-apikit/src/main/java/org/mule/module/apikit/UrlUtils.java

```diff
@@ -89,9 +91,20 @@ public static String getBasePath(MuleMessage message)
         if (path == null)
         {
             path = message.getInboundProperty("http.listener.path");
-            if (path != null && path.endsWith("/*"))
-            {
-                path = path.substring(0, path.length() - 2);
+
+            if (path != null) {
+                if (path.endsWith("/*"))
+                {
+                    path = path.substring(0, path.length() - 2);
+                }
+
+                ParameterMap uriParams = message.getInboundProperty("http.uri.params");
```

Comment:
Isn't there a constant for `"http.uri.params"`?

Reply:
no